sbt_common/messaging/src/test/scala/uk/ac/wellcome/messaging/metrics/MetricsSenderTest.scala

```diff
@@ -10,6 +10,7 @@ import org.mockito.ArgumentCaptor
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
 import org.scalatest.mockito.MockitoSugar
 import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.monitoring.MetricsSender
```

Comment:
This test should be moved into the monitoring lib, and then you can get rid of this import.